worlds/dark_souls_3/docs/setup_en.md

```diff
@@ -22,7 +22,7 @@ version before using this version.
 ### One-Time Setup
 
 Before you first connect to a multiworld, you need to generate the local data files for your world's
-randomized item and (optionally) enemy locations. You only need to do this once per multiworld.
+randomized items and (optionally) enemy locations. You only need to do this once per multiworld.
```

Comment:
I think this is correct—the pluralization of "item" should match "enemy" ("locations" here refers to both).

Reply:
Oh, I see how it's being used now it's "item locations" and "enemy locations"